src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -838,6 +920,14 @@ private PreparedStatement getStreamFromDb(Connection conn,
     return st;
   }
 
+  private PreparedStatement getExtraMetadataFromDb(Connection conn,
```

Comment:
We're planning to rename these methods, so lets go with prepareExtraMetadataStatement for this one now, so the change doesn't collide with this one.